### PR TITLE
WorldMap enhancements

### PR DESCRIPTION
**Pull request by markdwags**
WorldMap reads CSV, UOAM and Ultima Mapper marker files from `Data\Client` -- *.map files and *.xml files -- and displays them on the map.

Updated context menu to show/hide specific markers
Markers only render if they are in the the viewport
Labels won't render when zoomed out at a certain level

### CODE REVIEW

**Comment by Alias-Alan** [comment]
This is an amazing add-in Quick, fantastic!! Would there be a way to import small icons(basic ones like moongate, staircase for dungeon levels, skullandcrossbones for dungeon entrances, a res shrine for shrines) that would be pretty cool. Love that you added the ability to color markers though!

File: src/Game/UI/Gumps/WorldMapGump.cs [hunk 1]
```diff
@@ -86,6 +121,13 @@ public override void Restore(XmlElement xml)
             if (int.TryParse(xml.GetAttribute("zoomindex"), out int value))
                 _zoomIndex = (value >= 0 && value < _zooms.Length) ? value : 4;
 
+            _showCoordinates = bool.Parse(xml.GetAttribute("showcoordinates"));
```
Comment (andreakarasho) [inline src/Game/UI/Gumps/WorldMapGump.cs]: bool.TryParse instead of bool.Parse 
or it will throw errors

Comment (markdwags) [reply src/Game/UI/Gumps/WorldMapGump.cs]: Ok, bool.Parse is used throughout the class -- you want all of them refactored?

Comment (andreakarasho) [reply src/Game/UI/Gumps/WorldMapGump.cs]: Yes just to avoid exceptions with retro compatibility

Comment (markdwags) [reply src/Game/UI/Gumps/WorldMapGump.cs]: Made those changes to the bool parsing.

**Comment by andreakarasho** [comment]
Can you solve conflicts?

**Comment by markdwags** [comment]
> Can you solve conflicts?

Done though not sure it went through since I don't have write access.

**Comment by markdwags** [comment]
It's messed up, and I hate git.

**Pull request closed without merging**